test/unit/lib/utils/gpDataMapper.js

```diff
@@ -4,23 +4,31 @@ const gpDataMapper = require('../../../../app/lib/utils/gpDataMapper');
 const expect = chai.expect;
 
 describe('GP Surgery Data Mapper', () => {
-  it('for a GP Surgery that has POMI data and a url will return the url', () => {
-    const gpData = { bookingSystem: { supplier: 'EMIS', bookOnlineLink: 'http://example.com' } };
+  it('for a GP Surgery that has appointments POMI data and a url will return the url', () => {
+    const gpData = { onlineServices: { appointments: { supplier: 'EMIS', url: 'http://example.com' } } };
     const expectedResult = 'http://example.com';
     const result = gpDataMapper(gpData);
 
     expect(result).to.be.equal(expectedResult);
```

Comment:
Hmm, I've followed the same pattern here: https://github.com/nhsuk/gp-finder/blob/25a6678da426f44e64f31d1d522cd856f7ae5744/test/unit/lib/utils/gpDataMapper.js precisely because I believe it's more readable (note the change of mocha function)

How stongly do you feel about it? ;)